fastdds_python/test/api/test_datawriter.py

```diff
@@ -9,6 +9,68 @@ def __init__(self):
         super().__init__()
 
 
+def test_dispose():
+    """
+    This test checks:
+    - DataWriter::dispose
+    - DataWriter::dispose_w_timestamp
+    - DataWriter::unregister_instance
+    - DataWriter::unregister_instance_w_timestamp
```

Comment:
`unregister_instance_w_timestamp` does not seem to be called. It is called on `test_register_instance`, so I think this comment can be removed.